libs/langchain/langchain/agents/agent_toolkits/vectorstore/base.py

```diff
@@ -67,6 +122,55 @@ def create_vectorstore_router_agent(
 ) -> AgentExecutor:
     """Construct a VectorStore router agent from an LLM and tools.
 
+    Note: this class is deprecated. See below for a replacement that uses tool
```

Comment:
would explicitly callout that its using langgraph, you need to have langgraph installed

Reply:
thanks, done here and elsewhere